setup.py

```diff
@@ -1,7 +1,11 @@
 from setuptools import setup, find_packages
 
 from byudml import __version__
+from byudml import __imputer_path__
+from byudml import __metafeature_path__
 
+# make sure this is a valid entrypoint
+assert "d3m.primitives.metalearning.metafeature_extractor.BYU" == ("d3m.primitives." + ".".join(__metafeature_path__.split(".")[2:]))
```

Comment:
Having this line means that the python path is no longer single sourced.

Reply:
Oh, that was debugging code.  Let me take that out